style.less

```diff
@@ -1,111 +1,127 @@
-@orange:#F2994A;
-@purple:#BB6BD9;
-@light-blue:#56CCF2;
+@white: #f2f2f2;
+@orange: #F2994A;
+@purple: #BB6BD9;
+@light-blue: #56CCF2;
+@gray-dark: #4F4F4F;
 body {
-  margin:0;
-  padding:0;
-  position:absolute;
-  left:0;
-  top:0;
-  width:100%;
-  height:100%;
+    margin: 0;
+    padding: 0;
+    position: absolute;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
 }
+
 .page {
-  overflow:hidden;
-  position:absolute;
-  left:0;
-  top:0;
-  margin:0;
-  width:100%;
-  height:100%;
-  //width:384px;
-  //height:640px;
-  background:#f2f2f2;
-  //margin:auto;
-  //margin-top:10%;
-  font-size:18px;
-  overflow-y:scroll;
-  font-family: 'Taviraj', serif;
-  
-  &.landscape{
-    //width:640px;
-    //height:384px;
-  }
+    overflow: hidden;
+    position: absolute;
+    left: 0;
+    top: 0;
+    margin: 0;
+    width: 100%;
+    height: 100%;
+    //width:384px;
+    //height:640px;
+    background: #f2f2f2;
+    //margin:auto;
+    //margin-top:10%;
+    font-size: 18px;
+    overflow-y: scroll;
+    font-family: 'Taviraj', serif;
+    &.landscape {
+        //width:640px;
```

Comment:
Why are these commented out? is the idea to design a landscape layout in future?